backend/serverless.yml

```diff
@@ -40,17 +40,38 @@ resources:
     appReviewsTable:
       Type: AWS::DynamoDB::Table
       Properties:
-        TableName: appReviewsTable-${self:provider.stage}
+        TableName: reviewsTable-${self:provider.stage}
         AttributeDefinitions:
-          - AttributeName: id
+          - AttributeName: appId
             AttributeType: S
-          - AttributeName: store
+          - AttributeName: versionDate
             AttributeType: S
+          - AttributeName: compSentiment
+            AttributeType: N
         KeySchema:
-          - AttributeName: id
+          - AttributeName: appId
             KeyType: HASH
-          - AttributeName: store
+          - AttributeName: versionDate
```

Comment:
I thought we were using appId + store as our key? Let me know if I'm misunderstanding this syntax.

Reply:
You're right, I missed adding the second part of the key